### DISCUSSION TITLE
Is it possible to split the serverless function of vercel into multiple trunks.

### DISCUSSION THREAD

**Discussion by DemoMacro** [nitrojs/nitro#172] [General]
When I deploy nitro on vercel, I have a situation where the size of the aws lambda exceeds the limit, is it possible to split it into multiple functions?

**Comment by pi0** [comment]
Not at the moment. Nitro output is a single multi-chunk entry with lazy loading to leverage maximum shared chunk saving. May I ask what made your bundle size exceed [50/250MB limitation](https://vercel.com/docs/concepts/limits/overview#serverless-function-size)?

**Reply by DemoMacro** [reply]
I use `geoip-lite`, a module that automatically downloads a database of geolocation IP addresses ...... It is hard to imagine that such a database could be packed into a single chunk.
